cmd/pdbinspect/main.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/provider"
+	"storj.io/storj/pkg/storage/meta"
+)
+
+func main() {
+	ctx := context.Background()
+	var port string
+	var apiKey string
+	var prefix string
+	var endBefore string
+	var startAfter string
+	var recursive bool
+	var limit int
+	var metaFlags uint32
+	var jsonBool bool
```

Comment:
nit - I don't think the type needs to be in the variable. I would prefer any of the following over `jsonBool`: `json`, `jsonEnabled`, `jsonPrint`